spring-kafka/src/main/java/org/springframework/kafka/retrytopic/EndpointCustomizer.java

```diff
@@ -31,19 +30,15 @@
  * @see EndpointCustomizerFactory
  *
  */
-interface EndpointCustomizer extends Function<MethodKafkaListenerEndpoint<?, ?>,
-		Collection<EndpointCustomizer.TopicNamesHolder>> {
+@FunctionalInterface
+interface EndpointCustomizer {
```

Comment:
Why this is still not `public`?

Can you move your tests to different package to see the problem?

Reply:
Sorry again, got distracted by the other changes. Fixed it now.